bin/generate-api-doc.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const fs = require('fs')
+const map = require('lodash.map')
+const flow = require('lodash.flow')
+const sortBy = require('lodash.sortby')
```

Comment:
Not a big deal as its for development here, but I've typically opted for just `require('lodash/{module}')` - I know that can add a lot to the final module size if that's a concern but I believe there's other ways around that if it's a major concern as well.